packages/amplify-cli-core/src/__tests__/packageManager.test.ts

```diff
@@ -63,3 +63,17 @@ describe('packageManager tests', () => {
     expect(packageManager!.packageManager).toEqual('npm');
   });
 });
+
+describe('Descope package Name', () => {
```

Comment:
after moving `descopePackageName` into `getAmplifyPackageLibDirPath` these tests shouldn't be necessary anymore